conf/application.conf

```diff
@@ -16,7 +16,7 @@ include "frontend.conf"
 
 appName = "customer-advisors-frontend"
 
-play.i18n.langs = ["en", "cy"]
+play.i18n.langs = ["en"]
```

Comment:
We are removing this as it is used by internal user and we do not need Welsh translation?